.gitlab-ci.yml

```diff
@@ -718,8 +717,7 @@ zombienet-tests-parachains-disputes:
     - /home/nonroot/zombie-net/scripts/run-test-env-manager.sh
         --github-remote-dir="${GH_DIR}"
         --test="0002-parachains-disputes.feature"
-  allow_failure:                   true
-  retry: 2
```

Comment:
why remove the retries?

Reply:
To ensure possible issues are not hidden by retries.

Reply:
Well, if the issues are resolved by retries, are they not most likely spurious?

Reply:
Yeah, but the assumption is now that these shouldn't fail and if it happens,  some level of scrutiny is needed to see if the test is flaky or there is an underlying issue.